ct-app/core/components/hoprd_api.py

```diff
@@ -290,6 +290,10 @@ async def ticket_price(self) -> Optional[float]:
 
         return float(response.price) / 1e18 if hasattr(response, "price") else None
 
+    async def winning_probability(self) -> Optional[float]:
+        # TODO: update to an API call once the endpoint is available
+        return 1
```

Comment:
call will be simpler than the proposed piece of code. It will look like: 
```python
_, response = await self.__call_api(sdk.api.NetworkApi, "winning_probability")
return response.probability       
```
which is similar to the `ticket_price` endpoint. As it's this simple, nothing has to be changed for now.